libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/view/MapboxManeuversList.kt

```diff
@@ -0,0 +1,36 @@
+package com.mapbox.navigation.ui.maneuver.view
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.util.AttributeSet
+import androidx.recyclerview.widget.RecyclerView
+
+/**
+ * Class extending [RecyclerView] to intercept request layout.
+ */
+class MapboxManeuversList @JvmOverloads constructor(
```

Comment:
No. This is a public class extending `RecyclerView` that is being used in the layout file instead of using the `RecyclerView`. I am not a big fan of this, but had to do it to resolve the error you can see I have added as a comment in this class. This is a known issue in android.

Reply:
On the contrary, if it's `public` clients could think that this a standalone component as the rest of the views ¯\\\_(ツ)\_/¯

Reply:
Done.